tests/testthat/test-markdown.R

```diff
@@ -63,3 +63,8 @@ test_that("validates math yaml", {
     config_math_rendering_(`math-rendering` = "math")
   })
 })
+
+test_that("preserves ANSI characters", {
+    pkg <- local_pkgdown_site()
+    markdown_text(pkg, "prefer [1m[34mdplyr[39m[22m::filter")
```

Comment:
The string needs `'\033'`, as in:

``` r
options(cli.num_colors = 256)
dput(cli::col_blue("a"))
#> structure("\033[34ma\033[39m", class = c("cli_ansi_string", "ansi_string", 
#> "character"))
```

<sup>Created on 2024-10-03 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>

Reply:
Very helpful, thank you!